includes/data-stores/class-wc-admin-reports-downloads-data-store.php

```diff
@@ -0,0 +1,375 @@
+<?php
+/**
+ * WC_Admin_Reports_Downloads_Data_Store class file.
+ *
+ * @package WooCommerce Admin/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Reports_Downloads_Data_Store.
+ */
+class WC_Admin_Reports_Downloads_Data_Store extends WC_Admin_Reports_Data_Store implements WC_Admin_Reports_Data_Store_Interface {
+
+	/**
+	 * Table used to get the data.
+	 *
+	 * @var string
+	 */
+	const TABLE_NAME = 'wc_download_log';
+
+	/**
+	 * Mapping columns to data type to return correct response types.
+	 *
+	 * @var array
+	 */
+	protected $column_types = array(
+		'id'          => 'intval',
+		'date'        => 'strval',
+		'date_gmt'    => 'strval',
+		'download_id' => 'strval', // String because this can sometimes be a hash.
+		'file_name'   => 'strval',
```

Comment:
Should some of these fall into the `extended_info` area?

Reply:
I don't think so. All of these fields are columns in the report table, so I think we will always want them present in the request.